src/platform-apps/channels/channel-view-container.tsx

```diff
@@ -29,25 +33,43 @@ export class Container extends React.Component<Properties> {
   static mapActions(_props: Properties): Partial<Properties> {
     return {
       fetchMessages,
+      startMessageSync,
     };
   }
 
+  state = { countNewMessages: 0 };
+
   componentDidMount() {
     const { channelId } = this.props;
 
     if (channelId) {
       this.props.fetchMessages({ channelId });
+      this.props.startMessageSync({ channelId });
```

Comment:
i think we probably also need to stop the message sync, when the user switches channels, or navigates away from channels entirely. it's possible that the saga just handles the first scenario (we should verify that if we haven't), but in the case of navigating away from channels, we need to have an action that stops the channel syncing, or it will continue in the background.